docs/conf.py

```diff
@@ -159,28 +145,30 @@ def setup(app):
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-    ('contents', 'malcolm.tex', u'malcolm Documentation',
-     u'Tom Cobb', 'manual'),
+    ("contents", "malcolm.tex", u"malcolm Documentation", u"Tom Cobb", "manual"),
```

Comment:
This can all go, in fact you could replace most of this file with conf.py from the template module